src/plugins/sync-plugin.js

```diff
@@ -252,7 +255,7 @@ export class ProsemirrorBinding {
   unrenderSnapshot () {
     this.mapping = new Map()
     this.mux(() => {
-      const fragmentContent = this.type.toArray().map(t => createNodeFromYElement(/** @type {Y.XmlElement} */ (t), this.prosemirrorView.state.schema, this.mapping)).filter(n => n !== null)
+      const fragmentContent = this.type.toArray().map(t => createNodeFromYElement(/** @type {Y.XmlElement} */(t), this.prosemirrorView.state.schema, this.mapping)).filter(n => n !== null)
```

Comment:
All of the edits from here down are a result of running `standard --fix` before committing vvv

Reply:
I care about these things more than I should. I use the space consistently between projects, so I need to reverse your changes every time I merge your PRs.. (small price because your PRs are much appreciated)

The package wouldn't publish if that was not validly formatted. I use standard v16 (the current version). This package uses v12, which also doesn't define that rule.

For the future, you could just run `npm run lint` using the intended version.

Reply:
>  I need to reverse your changes every time I merge your PRs..

I'm not _trying_ to change anything, I'm just running the included linters. It would be very useful if you'd consider setting up CI on these projects, this is the only project I contribute to that doesn't enforce linting rules automatically, it would save your time in the long run I'm sure.

Reply:
You are not running standard@v12. And a CI can't fix that. In the future, instead of using your linter, I'm just asking you to run `npx standard --fix` using the shipped version of standard.

I would be very surprised if the linter works differently on your computer. Maybe it does. Maybe you setup some custom rules. Then please make my life easier by fixing the linting manually.

Your code is technically valid `standard` code. So a CI can't enforce you not to run your linter. 

I'm all for setting up CI. For different reasons though.

Reply:
I don't want to seem like I'm arguing, I just want to be clear – I'm absolutely running `standard@v12`, it's installed in `node_modules` as per the package.json and running `npx standard --fix` from the command line results in this exact same output 🤷

Reply:
It seems to be related to your system. I'm running these packages on Linux, Mac, and Windows with virtually any Node version (10, 12, and 14). Here, I also added travis CI to this project:

[![Build Status](https://travis-ci.com/yjs/y-prosemirror.svg?branch=master)](https://travis-ci.com/yjs/y-prosemirror)

The project is linted correctly, and I'd like not to change this space everywhere. In that case please don't run the --fix command. Maybe you can figure out the problem and report an issue to the `standard` project.